### PR TITLE
Tags css

### PR DESCRIPTION
**Pull request by clundin55**

### CODE REVIEW

File: yw_website/yw_website/apps/website/static/css/bulma.css [hunk 1]
```diff
@@ -0,0 +1,225 @@
...
+}
+
+.tag:not(body).is-rounded {
+  border-radius: 290486px;
+}
+
+a.tag:hover {
+  text-decoration: underline;
+}
+/*# sourceMappingURL=bulma.css.map */
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/static/css/bulma.css:225]: pretty sure you can just include the bulma css link in the header and use these

might be wrong but i'm pretty sure thats the case

Comment (aniehuser) [reply yw_website/yw_website/apps/website/static/css/bulma.css:225]: we can create custom classes to override them

File: yw_website/yw_website/apps/website/templates/includeable/tagarray.html [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<div class="tags has-addons">
+{% for tag in tags %}
+  <div class="control">
+    <div class="tags has-addons">
+      <a href="{% url 'home' %}?q={{ tag.tag.title }}" class="tag is-link">{{tag.tag.title}}{% if not forloop.last %}{% endif %}</a>
+      <a href="{% url 'tag_delete' tag.tag.id %}" class="tag is-delete"></a>
+    </div>
+  </div>
+{% endfor %}
+</div>
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/templates/includeable/tagarray.html:14]: good addition

File: yw_website/yw_website/apps/website/templates/includeable/tagarray.html [hunk 3]
```diff
@@ -0,0 +1,14 @@
+{% load static %}
+{% get_static_prefix as STATIC_PREFIX %}
+<link rel="stylesheet" type="text/css" href="{{ STATIC_PREFIX }}css/bulma.css">
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/templates/includeable/tagarray.html:3]: this is what i meant by the bulma css link
```
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/bulma/0.7.4/css/bulma.min.css">
```
it'll work both ways, up to you

File: yw_website/yw_website/apps/website/templates/includeable/tagarray.html [hunk 4]
```diff
@@ -0,0 +1,14 @@
...
+
+<div class="tags has-addons">
+{% for tag in tags %}
+  <div class="control">
+    <div class="tags has-addons">
+      <a href="{% url 'home' %}?q={{ tag.tag.title }}" class="tag is-link">{{tag.tag.title}}{% if not forloop.last %}{% endif %}</a>
```
Comment (aniehuser) [inline yw_website/yw_website/apps/website/templates/includeable/tagarray.html]: not sure why you have `{% if not forloop.last %}{% endif %}`

File: yw_website/yw_website/apps/website/static/css/bulma.css (hunk 1 above)
Comment (clundin55) [reply yw_website/yw_website/apps/website/static/css/bulma.css:225]: Maybe, I do know that a lot of CSS collides with the Bulma css found in this file and messes up our other elements, so this is how I solved it

Comment (clundin55) [reply yw_website/yw_website/apps/website/static/css/bulma.css:225]: Let me know what you think

File: yw_website/yw_website/apps/website/templates/includeable/tagarray.html (hunk 3 above)
Comment (clundin55) [reply yw_website/yw_website/apps/website/templates/includeable/tagarray.html:3]: See above comment

File: yw_website/yw_website/apps/website/templates/includeable/tagarray.html (hunk 4 above)
Comment (clundin55) [reply yw_website/yw_website/apps/website/templates/includeable/tagarray.html]: Oops relic of how I was adding commas

**Review by aniehuser — DISMISSED** [review]
A few small considerations, but this is good stuff very nice

File: yw_website/yw_website/apps/website/templates/includeable/tagarray.html (hunk 4 above)
Comment (aniehuser) [reply yw_website/yw_website/apps/website/templates/includeable/tagarray.html]: lol

File: yw_website/yw_website/apps/website/static/css/bulma.css (hunk 1 above)
Comment (aniehuser) [reply yw_website/yw_website/apps/website/static/css/bulma.css:225]: if their was in issue screw it, leave as is

**Comment by clundin55** [comment]
@aniehuser removed the if statement, if you could re-approve. thanks.

**Review by aniehuser — APPROVED** [review]

**Pull request merged**